src/apps/opening-hours-editor/DialogFomularEdit.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { EventImpl } from "@fullcalendar/core/internal";
+import {
+  adjustEndDateBasedOnStartDate,
+  extractTime,
+  formatFullCalendarEventToCmsEvent,
+  updateDateTime
+} from "./helper";
+import EventForm, { EventFormOnSubmitType } from "./EventForm";
+import { useText } from "../../core/utils/text";
+import { OpeningHoursCategoriesType } from "./types";
+import { DplOpeningHoursListGET200Item } from "../../core/dpl-cms/model";
+
+type DialogFomularEditProps = {
+  eventInfo: EventImpl;
+  handleEventEditing: (event: DplOpeningHoursListGET200Item) => void;
+  closeDialog: () => void;
+  handleEventRemove: (eventId: string) => void;
+  openingHoursCategories: OpeningHoursCategoriesType[];
+};
+
+const DialogFomularEdit: React.FC<DialogFomularEditProps> = ({
+  eventInfo,
+  handleEventEditing,
+  closeDialog,
+  handleEventRemove,
+  openingHoursCategories
+}) => {
+  const t = useText();
+
+  const handleSubmit: EventFormOnSubmitType = (
+    category,
+    startTime,
+    endTime
+  ) => {
+    if (!eventInfo.start || !eventInfo.end) {
+      // eslint-disable-next-line no-alert
+      alert("Invalid event");
```

Comment:
:+1: I think it is OK that we use simple alerts for error handling at the moment.

Consider making the text translatable since it is editor facing.

Also I wonder if it would make sense to log more detailed information about the error state.